core/api/src/lib.rs

```diff
@@ -59,7 +59,7 @@ impl Query {
 
     #[graphql(name = "getTransaction", description = "Get the transaction by hash")]
     async fn get_transaction(state_ctx: &State, tx_hash: Hash) -> FieldResult<SignedTransaction> {
-        let hash = protocol::types::Hash::from_hex(&tx_hash.as_hex())?;
+        let hash = protocol::types::Hash::from_hex(&tx_hash.as_hex()?)?;
```

Comment:
The input forces 0x, so it is recommended that output also starts with 0x.
It is recommended to modify the output hex of the `from_hex` method to start with 0x

Reply:
done

Reply:
please check again